website/static/js/contribManager.js

```diff
@@ -49,20 +49,30 @@ var setupEditable = function(elm, data) {
     });
 };
 
-var ContributorModel = function(contributor, pageOwner, isRegistration) {
+var ContributorModel = function(contributor, manager, pageOwner, isRegistration, isAdmin) {
```

Comment:
Perhaps `contributorsVm` instead of `manager` to clarify what the argument is supposed to be? Also, is the `pageOwner` argument necessary, since we can access the `user()` from the `ContributorsViewModel`?